practices/practice-one/src/index.ts

```diff
@@ -0,0 +1,9 @@
+import TodoFormView from "./scripts/views/todo-form.view";
+const todoFormView = new TodoFormView()
+
+if (document.readyState === 'loading') {
+    document.addEventListener('DOMContentLoaded', todoFormView.init)
+}
+else {
```

Comment:
Look like your config not work correctly, could you re-check it?

Reply:
- I still use it for other projects. I see it still works;
- Can you tell me more?

Reply:
Please in condition https://github.com/airbnb/javascript#blocks